### PR TITLE
Add react native paper & enhance UI for all views

### PR DESCRIPTION
**Pull request by Clebiez**
## Description

Enhance UI by adding React native paper.

## ToDo list:

- [x] install react native paper
- [x] Create a custom theme by using custom colors
- [x] Each views (login, home, game/:id & lobby) use now react native paper components. 
- [x] Use `date-fns` to handle date from unix timestamp & return well-formatted date
 
RNstyle

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,41 +1,89 @@
...
 
 import LoginScreen from "./src/screens/Login.jsx";
 import HomeScreen from "./src/screens/Home.jsx";
 import GameScreen from "./src/screens/Game.jsx";
 import LobbyScreen from "./src/screens/Lobby.jsx";
+import { screensEnabled } from "react-native-screens";
 
 const Stack = createStackNavigator();
 
+const CombinedDarkTheme = {
```
Comment (alexisjanvier) [inline App.js]: this could be isolated in a theme file?

File: src/components/Title.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import { StyleSheet } from "react-native";
+import { useTheme, Title as RNPTitle } from "react-native-paper";
```
Comment (alexisjanvier) [inline src/components/Title.jsx]: `RNPTitle` ? What does that mean?

Comment (guilbill) [inline src/components/Title.jsx]: Maybe PaperTitle like on the react-native-paper import on App.js ?

**Review by guilbill — APPROVED** [review]

File: src/components/Title.jsx (hunk 2 above)
Comment (Clebiez) [reply src/components/Title.jsx]: As suggested by @guilbill I just move this to `PaperTitle`

**Review by alexisjanvier — APPROVED** [review]

**Pull request merged**